packages/components/src/internal/components/search/EntityFieldFilterModal.tsx

```diff
@@ -286,41 +322,46 @@ export const EntityFieldFilterModal: FC<Props> = memo(props => {
                                             </Tab.Pane>
-                                            {/* <Tab.Pane eventKey={EntityFieldFilterTabs.ChooseValues}>*/}
-                                            {/*    <div className="parent-search-panel__col-sub-title">Find values for {activeField.caption}</div>*/}
-                                            {/*    {*/}
-                                            {/*        activeField?.allowFaceting() &&*/}
-                                            {/*        <FilterFacetedSelector*/}
-                                            {/*            selectDistinctOptions={*/}
-                                            {/*                {*/}
-                                            {/*                    column: activeField?.fieldKey,*/}
-                                            {/*                    // containerFilter: model.containerFilter,*/}
-                                            {/*                    // containerPath: model.containerPath,*/}
-                                            {/*                    schemaName: entityDataType?.instanceSchemaName,*/}
-                                            {/*                    queryName: activeQuery,*/}
-                                            {/*                    viewName: "",*/}
-                                            {/*                    filterArray: [], //TODO use active filters to filter distinct values, but exclude filters on current field*/}
-                                            {/*                    parameters: null, //TODO use active parameters to filter distinct values*/}
-                                            {/*                }*/}
-                                            {/*            }*/}
-                                            {/*        />*/}
-                                            {/*    }*/}
-                                            {/* </Tab.Pane>*/}
+                                            {activeTab === EntityFieldFilterTabs.ChooseValues && allowFaceting && (
+                                                <Tab.Pane eventKey={EntityFieldFilterTabs.ChooseValues}>
+                                                    <div className="parent-search-panel__col-sub-title">
+                                                        Find values for {activeField.caption}
+                                                    </div>
+                                                    <FilterFacetedSelector
+                                                        selectDistinctOptions={{
+                                                            column: activeField?.fieldKey,
+                                                            schemaName: entityDataType?.instanceSchemaName,
+                                                            queryName: activeQuery,
+                                                            viewName: '',
```

Comment:
what does the empty string for the viewName mean here? is the default for that `selectDistinctOptions` prop something else or could this empty string become the default within `FilterFacetedSelector`?